src/main/steps/c100-rebuild/legal-representation-application/template.njk

```diff
@@ -0,0 +1,5 @@
+{% extends "common/template.njk" %}
```

Comment:
@niranjan-301 Can we move 'legal-representation-application' inside 'screening-questions' as it's part of screening questions feature.